gvm

```diff
@@ -2,7 +2,7 @@
 
 ###############################################################################
 # GENERATED VERSION! DO NOT CHANGE!                                           #
-SCRIPTVERSION="local-dev"
+SCRIPTVERSION="latest"
```

Comment:
Can you update this to `local-dev`? The script version should be `local-dev` for changes. It's updated by the CI/CD pipeline to match the tagged version prior to generating the sum file for comparison on update.

Reply:
No problem.  Just updated in latest commit.